dir attribute typo (#40130)
mdn/content: files/en-us/web/html/reference/global_attributes/dir/index.md

This attribute is _not_ inherited by the {{ HTMLElement("bdi") }} element. If not set, its value is `auto`.

Browsers might allow users to change the directionality of {{ HTMLElement("input") }} and {{ HTMLElement("textarea") }} elements in order to assist with authoring content.
Chrome and Safari provide a directionality option in the contextual menu of input fields.

> Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction and also updates the `dir` attribute value.

Author: ```suggestion
Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction, and also updates the `dir` attribute value.
```

> Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction, and also update the `dir` attribute value.

Reviewer: ```suggestion
Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction.
In all these browsers, the `dir` attribute is automatically updated.
```

Reviewer 2: I think it is right to put this bit in a separate sentence, but I also think it needs to be linked to the described browser features a bit more, and we need to make it sound less like the `dir` attribute update is an additional/adjacent effect; it is how the text directionality is being changed.

```suggestion
Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction.
These features toggle the `dir` attribute value between `ltr` and `rtl`.
```

Author: Thanks. Selecting the second option. Both better than what I had.

> Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction, and also updates the `dir` attribute value.

Author: ```suggestion
Firefox uses <kbd>Ctrl</kbd> (Windows)/<kbd>Cmd</kbd> (macOS) + <kbd>Shift</kbd> + <kbd>X</kbd> inside a `<textarea>` to toggle text direction, and also update the `dir` attribute value.
```
